packages/website/docs/_components_pages/main/Text.mdx

```diff
@@ -21,7 +24,22 @@ The **maxLines** property defines the number of lines the text would wrap before
 
 <MaxLines />
 
+### Empty Indicator Mode
+The **emptyIndicatorMode** property specifies if an empty indicator should be displayed when there is no text.
+
+<EmptyIndicatorMode />
+
 ### Custom Styling
 The Text can be customized with pure CSS, applied on the tag.
 
-<CustomStyling />
\ No newline at end of file
+<CustomStyling />
+
+### Render white space with custom styling
+How white space is handled in the Text can be set with pure CSS, applied on the tag.
```

Comment:
maybe:
How a white space is handled if the text can be set with pure CSS, applied on the tag.